databricks/koalas/tests/test_dataframe.py

```diff
@@ -1040,3 +1040,15 @@ def test_rank(self):
         msg = "method must be one of 'average', 'min', 'max', 'first', 'dense'"
         with self.assertRaisesRegex(ValueError, msg):
             kdf.rank(method='nothing')
+
+    def test_shift(self):
+        pdf = pd.DataFrame({'Col1': [10, 20, 15, 30, 45],
+                            'Col2': [13, 23, 18, 33, 48],
+                            'Col3': [17, 27, 22, 37, 52]})
+        kdf = ks.from_pandas(pdf)
+        self.assert_eq(pdf.shift(3), kdf.shift(3).sort_index())
+        self.assert_eq(pdf.shift(periods=3, fill_value=0),
+                       kdf.shift(periods=3, fill_value=0).sort_index())
```

Comment:
@RainFung, you can take one of three usually in this case.

1. Check pandas version via `distutils.version.LooseVersion` and skip the test conditionally with `unittest.skipIf`.
2. Creates a string or pre-defined dataframe and compare with `kdf` here. For instance, you can do:
    ```python
    # Pandas 0.23.x does not have 'fill_value' parameter. Therefore, here we do a manual comparison.
    pdf = pd.DataFrame({'Col1': [0, 0, 0, 10, 20],
                        'Col2': [0, 0, 0, 13, 23],
                        'Col3': [0, 0, 0, 17, 27]})
    self.assert_eq(pdf, kdf.shift(periods=3, fill_value=0).sort_index())
    ```

3. Simply don't add this tests if it looks obviously fine to skip.